### DISCUSSION TITLE
Cockpit View Looks Different when in Liberation

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#1756] [Q&A] [answered]
Hi, when I fly a mission in Liberation, I find my Hornet cockpit default view proportions look different than when I am not using Liberation. See attached screenshot comparison. It's like the seat is too high and I am looking over at the aircraft front. And the front panels look like they have been stretched wider. Adjusting the seat height helps but doesn't quite give me back the default view that what I have when I am not flying Liberation. Has it got to do with some adjustments that occurred at catapult launch? Or was the FOV somehow edited in the campaign? Thanks 
![Clipboard](https://user-images.githubusercontent.com/94265170/141815171-06422111-2bb5-412f-8cf0-7afc0dd9f8ff.jpg)
.

**Comment by peteauau** [comment]
looks like  its  a vr  o  tr  view  issues   from  your  previous  set up

**Comment by Starfire13** [comment]
My view in Liberation looks pretty much like the one on the left. Liberation doesn't do anything to view angle or fov, so any changes are on your end.

**Comment by GrEaSeLiTeNiN** [comment, marked as answer]
I was using TrackIR  Z-axis for View Zoom. Changing it back to the default Head Forward-Backward solved this. So what was happening is that the view on the right was a Head Forward view that seemed to be stuck.
